examples/crossplatform.pp

```diff
@@ -0,0 +1,89 @@
+case $facts['os']['family'] {
```

Comment:
Yep, see the docs for the puppet facts gem these tests use https://github.com/mcanevet/rspec-puppet-facts and their example of a case statement
```
# If you need any to specify any operating system specific tests
      case facts[:osfamily]
      when 'Debian'
        ...
      else
        ...
      end
```